src/components/Button/Button.component.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import "./Button.style.css";
+import Polygon from "../ImageComponents/FatArrow.icon";
```

Comment:
When you import a component, in most cases, the import should be named the same as the component.